basic_auth/put_user.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+import argparse
+import getpass
+import json
+import secrets
+from dataclasses import asdict
+
+import boto3
+
+from handler import User, hash_password, region
+
+
+def main():
+    parser = argparse.ArgumentParser()
```

Comment:
It might be nice to build the `ArgumentParser` in a seperate function, so that external tools like [sphinx-argparse](https://sphinx-argparse.readthedocs.io/en/stable/index.html) can access them.